docker-storage-setup.sh

```diff
@@ -1049,8 +1049,10 @@ get_docker_root_dir(){
                 ;;
         esac
     done
-    if ! DOCKER_ROOT_DIR=$(realpath -m $path);then
-      Fatal "realpath failed on $path"
+    if [ ! -z "$path" ];then
```

Comment:
Won't this end up with DOCKER_ROOT_DIR not defined.

Why not just do 
 if [ ! -z "$path" ];then
          path = $DEFAULT_CONTAINER_PATH
fi